doc/command_ref.rst

```diff
@@ -1446,6 +1446,11 @@ Files
 
 .. _see_also-label:
 
+========
+Footnotes
+========
+DNF cannot import, list or remove public GPG keys (gpg-pubkey-* packages) because they aren't real packages and they are not stored in the RPM database. Use the :manpage:`rpm(8)` and :manpage:`rpmkeys(8)` commands for that.
```

Comment:
I'm not sure about wording. I think despite the fact that these packages are not "real", they are still stored in the rpmdb.

Reply:
What about:
```DNF do not import information about public GPG keys (gpg-pubkey-*) therefore it cannot list or remove them.```

Reply:
But that's not true either, because DNF *does* import them, via gpgkey= in repo config.

An alternative approach to just denying it all three times might be adding 'dnf import-key' or such to the cli ;)

Reply:
...and on that note, you could just as well do import-keys, list-keys and remove-keys as special operations that talk directly to librpm and bypass libsolv entirely, as for sure *libsolv* isn't interested in dealing with gpg-pubkeys because there's nothing to "solve" there - they are not packages although they look like it a bit.

Reply:
We will think about

Reply:
I'd rather see this as part of the core, but I like the idea of `dnf key <subcommand>`...